build.gradle

```diff
@@ -369,4 +369,15 @@ tasks.register("listRepositories") {
         println "Repositories:"
         project.repositories.each { println "Name: " + it.name + "; url: " + it.url }
    }
-}
\ No newline at end of file
+}
+
+tasks.register("githubBuildDetails", GithubDetailsTask) {
+    text = """### Test result files
+
+- [Unit Tests]({reportsUrl}/tests/testReleaseUnitTest/index.html)
+- [Android Lint]({reportsUrl}/lint-results-release.html)"""
+}
+
+tasks.register("githubLintDetektDetails", GithubDetailsTask) {
+    text = "### [Detekt Results](/reports/detekt.html)"
```

Comment:
You forgot `{reportsUrl}`. But even then, I don't think `https://firefoxci.taskcluster-artifacts.net/{reportsUrl}/0/public/reports/detekt.html` is a correct url (or at least, it doesn't work for me).

Also, this url renders as a sub-header in markdown, probably not what you want.